compiler/test/test_end_to_end.re

```diff
@@ -162,6 +162,7 @@ let basic_functionality_tests = [
   t("complex2", "print(2 + 3)", "5\nvoid"),
   t("binop1", "2 + 2", "4"),
   t("binop2", "2 - 2", "0"),
+  t("binop2.1", "2-2", "0"),
```

Comment:
Please add tests for the cases with newlines which we discussed on Discord (and a reminder about opening that issue).